server/publicapi/features/prepopulate.feature

```diff
@@ -7,7 +7,7 @@ Feature: Prepopulate public content
     	"""
         Then we get response code 201
 
-        When we get "/items"
+        When we get "/items?start_date=1970-01-01"
```

Comment:
As per the current specification, this is not supported, but I agree it would make sense. The reason we opted for dates only was that many APIs from the major news companies behave the same.

@hlmnrmr what do you think about it? When we were discussing this, we didn't know the exact reason for that, we just followed their support for dates only.